packages/angular/generators.json

```
@@ -80,6 +80,11 @@
       "aliases": ["mv"],
       "description": "Moves an Angular application or library to another folder within the workspace and updates the project configuration."
     },
+    "move-to-with-mf": {
```

I'm definitely not an English native speaker, so I might be wrong, but I think `convert` fits better what this does, wdyt?
```suggestion
    "convert-to-with-mf": {
```

---

yeah i think you're right